.chloggen/builder-strict-versioning.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: builder
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add strict version checking when using the builder and the flag --skip-strict-versioning for skipping.
```

I want to make clear the flag is temporary in the release note as well

```suggestion
note: Add strict version checking when using the builder. Add the temporary flag `--skip-strict-versioning `for skipping this check.
```